openfl/utilities/utils.py

```diff
@@ -236,3 +236,22 @@ def merge_configs(
             settings.set(key, operation(value))
     settings.validators.validate()
     return settings
+
+
+def change_tags(tags, add_field=None, remove_field=None) -> Tuple[str]:
+    """Change tensor tags to add or remove fields.
+
+    Args:
+        tags(tuple): tensor tags.
+        add_field(str): add a new tensor tag field.
+        remove_field(str): remove a tensor tag field.
+    """
+    tags = list(tags)
+
+    if add_field:
+        tags.append(add_field)
+    if remove_field:
+        tags.remove(remove_field)
+    # remove duplicates while keeping the order
+    tags = tuple(dict.fromkeys(tags))
```

Comment:
Should we keep order? Are tags ('a', 'b') and ('b', 'a') different? Probably we should use the same sorting algorithm instead keeping order, for example `tags = tuple(tags.sort()).